test/ingress/conformance/tests/httproute-http2rpc.yaml

```diff
@@ -0,0 +1,70 @@
+# Copyright (c) 2022 Alibaba Group Holding Ltd.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: networking.higress.io/v1
+kind: Http2Rpc
+metadata:
+  name: httproute-http2rpc-demo
+  namespace: higress-system
+spec:
+  routeType: "DUBBO"
```

Comment:
If use dubboService instead of routeService, this routeType is meaningless and can be removed.

Reply:
But if we use the same CRD to support gRPC and other service types, corresponding filter can use this `routeType` field to tell which fields shall be read, ( [dubboService, dubboMethod] vs [gRPCService, gRPCMethod], just in case that all of them are not empty.

Reply:
I think CRD should be as user-friendly as possible and reduce configuration complexity. In terms of implementation, you can judge whether it is the intention of HTTP2Dubbo based on the `dubboService` field.

Reply:
However, if user configures both dubboService and gRpcService, which one should the plugin use? Or can we separate this http2rpc CRD to http2dubbo and http2grpc CRDs for different scenarios?

Reply:
Like this:
```proto
message  http_2_rpc {
  oneof destination {
     string dubbo_service = 1;
     string grpc_service = 2;
  }
}
```
and the CRD will generated like this:
```yaml
http2Rpc:
  type: object
  properties:
    dubboService:
      type: string
    grpcService:
      type: string
  oneOf:
   - not:
         anyOf:
         - required:
            - dubboService
         - required:
            - grpcService
  - required:
    - dubboService  
  - required:
    - grpcService
```
In this way, k8s will ensure that only one of the fields `dubboService` and `grpcService` appears.